schemas/metaschema/DatabaseField.schema

```diff
@@ -1,17 +1,18 @@
 {
   domain: { domain: 'Nomen' },
   category: { domain: 'Nomen' },
-  required: { domain: 'Logical' },
+  required: { domain: 'Logical' }, // empty string is not considered as correct
```

Comment:
It may be better to add a `truthy` property to a record or treat `min` for `string` as a minimum length.

Reply:
Main purpose for schemas is "to be subject domain oriented", so `truthy: true` is clear for developer, but `required: true` is better for any analyst.

Reply:
What about using `min` for string, because an empty string is still a string? I don't like the idea of `require` working differently throughout types: the initial value for numbers (`0`) is valid, yet the initial value for strings (`''`) is not.

Reply:
Required will deny only: `undefined`, `null`, `not set`, `NaN`